### DISCUSSION TITLE
No email from Existential Audo

### DISCUSSION THREAD

**Discussion by toiletwrecker** [ExistentialAudio/BlackHole#564] [Q&A]
I don't really know where else to put this, nobody is answering me. I have signed up with four different emails on the existential audio website, and have been waiting for over an hour. I have checked all spam folders, and i have received absolutely nothing from this website. Did they stop providing this program? What is going on? I really want the installer, and I am very frustrated that I have not received anything. I have only been able to download the github zip file, and have not yet received the download link.
[README.md](https://github.com/ExistentialAudio/BlackHole/fil
[CHANGELOG.md](https://github.com/ExistentialAudio/BlackHole/files/8696246/CHANGELOG.md)
es/8696241/README.md)

**Comment by devinroth** [comment]
It appears that there are some issues with my email server. Working on it now.

But in the meantime use "brew install blackhole-2ch"

Devin

**Comment by toiletwrecker** [comment]
thank you, do i just do that in the command line interface? which app do i
use?

Thanks,
charles

**Comment by devinroth** [comment]
Yeah terminal. You’ll need to install Home-brew though.

https://brew.sh

Devin

**Comment by devinroth** [comment]
Email server is back up and running.

Devin

**Comment by toiletwrecker** [comment]
awesome, thanks for the help, and Im sorry i displayed my frustration.

thanks,
charles

**Comment by devinroth** [comment]
Apology accepted. Honestly I didn’t notice. Probably because I was more frustrated with my email server than anything else.

Cheers.

Devin
